bridge/src/signature.rs

```diff
@@ -0,0 +1,88 @@
+/// ECDSA signatures:
+/// conversion from/to byte vectors.
+/// from/to v, r, s components.
+
+use ethereum_types::H256;
+use ethabi;
+
+pub const SIGNATURE_LENGTH: usize = 65;
+
+/// an ECDSA signature consisting of `v`, `r` and `s`
+#[derive(PartialEq, Debug)]
+pub struct Signature {
```

Comment:
hah... something like this should be migrated to `ethereum-types` at some point ;)

Reply:
definitely : )